spring-cloud-app-broker-acceptance-tests/src/test/java/org/springframework/cloud/appbroker/acceptance/CloudFoundryAcceptanceTest.java

```diff
@@ -104,7 +106,10 @@
 	protected static final String BACKING_APP_PATH = "classpath:backing-app.jar";
 
 	@Autowired
-	protected CloudFoundryService cloudFoundryService;
+	protected AdminCloudFoundryService adminCloudFoundryService;
```

Comment:
Sorry, I'd like to rename this back to `BrokerCloudFoundryService` or something similar. It is the client that represents the broker, and doesn't have any admin authorities.

Also, can it be made `private`? I haven't checked to see all the use cases yet - just a thought. I'm thinking that all tests should be written from unprivileged user perspective rather than using the broker client, which wasn't an option before this PR.

Reply:
Sure, renamed it back to `BrokerCloudFoundryService`. Now thinking it could be not the best name since this service uses client id and secret provided via properties, which most likely is admin credentials and it does not represent the broker. So it is really `TestAdminCloudFoundryService`. But I think it is not an issue so happy to leave it `BrokerCloudFoundryService`. 

We still use this class in management tests, so can't yet make it `private`. I don't really see it a problem because these two classes have different methods, so it's not easy to mix them up.

Reply:
> this service uses client id and secret provided via properties, which most likely is admin credentials

Sorry @alek-sys I see what you mean now. I'd misunderstood which clients were in play here.  So there are three... the 'admin' client (used by tests) which is configured via properties, and then the 'broker' (used by the broker under test) and 'user' (used by the tests) clients which are configured dynamically in the test setup. Have I finally understood this properly? 😅 

So calling the 'admin' client the 'broker' client is misleading. Now I think you're right about `TestAdminCloudFoundryService`. Sorry, sorry...!

I think the most confusing thing is how we seem to flip between usage of the 'admin' and 'user' clients, e.g.

```
	protected Mono<String> manageApps(String serviceInstanceName, String serviceName,
		String planName, String operation) {
		return userCloudFoundryService
			.getServiceInstance(serviceInstanceName)
			.map(ServiceInstance::getId)
			.flatMap(serviceInstanceId ->
				adminCloudFoundryService
					.getApplicationRoute(testBrokerAppName())
					.flatMap(appRoute ->
```

but at least I think I understand why now. @royclarkson made a comment about merging the `admin` and `user` clients together - what do you think about this? Now that I'm understanding that these clients both represent the tests, and that the 'broker' client (the one actually representing the broker) does not have `cloud_controller.admin`, I quite like that idea. Would be less confusing? If the tests use their admin authority to access SI's in the user's org and space, in order to drive a broker which does _not_ have admin authority, are we okay with that? Or am I missing a problem?